apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```
@@ -1,48 +1,33 @@
 import { css, jsx } from '@emotion/react'
 import styled from '@emotion/styled'
-import {
-    Col,
-    notification,
-    Row,
-    RowProps,
-    Space,
-    Typography,
-} from 'antd'
+
+import { useIntl } from '@open-condo/next/intl'
```

ESLint

[error] `@open-condo/next/intl` import should occur after import of `@open-condo/next/auth`

(import/order)